jax/config.py

```diff
@@ -160,25 +159,14 @@ def parse_flags_with_absl(self):
           "see https://github.com/google/jax/blob/master/design_notes/omnistaging.md.\n"
           "To remove this warning, unset the JAX_OMNISTAGING environment variable.")
 
-  def register_omnistaging_disabler(self, disabler):
-    if self.omnistaging_enabled:
-      self._omnistaging_disablers.append(disabler)
-
   def enable_omnistaging(self):
-    if not self.omnistaging_enabled:
-      raise Exception("can't re-enable omnistaging after it's been disabled")
+    return  # TODO(mattjj): remove all callers
```

Comment:
I think a warning here would be appropriate.

Reply:
I'm going to attempt the change you suggest in an immediate follow-up PR. Specifically, I'm thinking something like:
1. get this PR in
2. make a pypi release
3. make disable_omnistaging an error, make enable_omnistaging at least a warning, possibly also removing the flag
4. make another pypi release